cli/cmd/service/set_role.go

```diff
@@ -0,0 +1,75 @@
+package service
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/rilldata/rill/cli/pkg/cmdutil"
+	adminv1 "github.com/rilldata/rill/proto/gen/rill/admin/v1"
+	"github.com/spf13/cobra"
+)
+
+func SetRoleCmd(ch *cmdutil.Helper) *cobra.Command {
+	var name, projectName, role string
+	setRoleCmd := &cobra.Command{
+		Use:   "set-role",
+		Short: "Set role for service",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			var roleOptions []string
+			if projectName != "" {
+				roleOptions = projectRoles
+			} else {
+				roleOptions = orgRoles
+			}
+
+			if ch.Interactive {
+				err := cmdutil.StringPromptIfEmpty(&name, "Enter service name")
+				if err != nil {
+					return err
+				}
```

Comment:
Consider making it a required positional arg for consistency with the other commands, e.g. `"edit <service-name>"` has it as a required positional arg.